binding-library/java/src/main/java/com/microsoft/azure/functions/signalr/annotation/SignalROutput.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.functions.signalr.annotation;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.ElementType;
+
+import com.microsoft.azure.functions.annotation.CustomBinding;
+
+/**
+ * <p>Output type to Azure SignalR Service</p>
+ */
+@Target({ ElementType.PARAMETER, ElementType.METHOD })
+@Retention(RetentionPolicy.RUNTIME)
+@CustomBinding(direction = "out", name = "", type = "SignalR")
+public @interface SignalROutput {
+    /**
+     * The variable name used in function.json.
+     * @return The variable name used in function.json.
```

Comment:
@anthonychu - Have you tested this? I would like to make sure there are no changes needed for this work on the worker.

Reply:
With help from Anthony, I've tested this. By replacing worker(build from dev branch) in the function core tools and latest maven plugin, it could work. Sample code can be found in this PR: [https://github.com/Azure/azure-functions-signalrservice-extension/pull/49/files](https://github.com/Azure/azure-functions-signalrservice-extension/pull/49/files)

---
In reply to: [259940994](https://github.com/Azure/azure-functions-signalrservice-extension/pull/48#discussion_r259940994) [](ancestors = 259940994)